packages/office-ui-fabric-react/src/components/Link/Link.styles.ts

```diff
@@ -0,0 +1,68 @@
+import {
+  getFocusStyle,
+  HighContrastSelector
+} from '../../Styling';
+import {
+  ILinkStyleProps,
+  ILinkStyles
+} from './Link.types';
+
+export const getStyles = (props: ILinkStyleProps): ILinkStyles => {
+  const { disabled, theme } = props;
+  const { palette } = theme;
```

Comment:
Just a general comment / question  - are there any places where a semantic slot can be used instead of colors from the palette?

Reply:
That's a good question, I had forgotten about the semantic slots. I'll see if we have any for Link.

Reply:
I've added semantic slots for Link, set these up in the default theme, and used them in the styles.